crates/nu-parser/src/parse_keywords.rs

```diff
@@ -1211,6 +1211,7 @@ pub fn parse_export_in_module(
     working_set: &mut StateWorkingSet,
     lite_command: &LiteCommand,
     module_name: &[u8],
+    module: &mut Module,
```

Comment:
It is not clear which module the `module` is. Is it the parent module? It would be useful to name it `parent_module`, or similar.

Reply:
Yeah!  I like the new name `parent_module`.  I fell into trouble to name it properly